dashboard/app/views/admin_styleguide/show.html.haml

```diff
@@ -0,0 +1,5 @@
+#styleguide-app
+%script{src: asset_path('js/marked/marked.js')}
```

Comment:
I tried to include `%link{href: asset_path('css/common.css'), rel: 'stylesheet', type: 'text/css'}` in here and I got a 404, but I think it is required for some of the apps components to look right (specifically the run button).

By the way, what else should I be including in here?